op-scim-bridge/templates/deployment.yaml

```diff
@@ -99,13 +99,33 @@ spec:
               value: "{{ .Values.scim.httpPort }}"
             - name: "OP_SESSION"
               {{- if .Values.scim.credentialsVolume }}
-              value: "/home/opuser/.op/{{ .Values.scim.credentialsVolume.file }}"
+              value: "/home/opuser/.op/{{ .Values.scim.credentialsVolume.files.scimFile }}"
               {{- end }}
-              {{- if .Values.scim.credentialsSecret }}
+              {{- if (.Values.scim.credentialsSecrets).scimsession }}
```

Comment:
The extra bracketing in these situations performs a double `if` on the same line: once for `.Values.scim.credentialsSecrets`, and another for `.Values.scim.credentialsSecrets.scimsession`. It's pretty elegant.